docs/misc/prune-example.txt

```diff
@@ -1,17 +1,29 @@
 borg prune visualized
 =====================
 
-Assume it is 2016-01-01, today's backup has not yet been made, you have
-created at least one backup on each day in 2015 except on 2015-12-19 (no
-backup made on that day), and you started backing up with borg on
-2015-01-01.
+To understand prune, it helps to substitute the terms `prune --keep` with `delete all backups except the latest backup of each of the last 'n' INTERVALS (secondly, minutely, hourly, daily, weekly, monthly, or yearly), including the present, that have backups. 
```

Comment:
`... (seconds, minutes, ...)`

because it explains the interval options.

Reply:
Agreed.  And I don't think there is any need to duplicate the information contained in `borg help prune`.  If needed, this part could point the reader there.